tun/Cargo.toml

```diff
@@ -11,12 +11,20 @@ socket2 = "0.4"
 tokio = { version = "1.28", features = [] }
 byteorder = "1.4"
 
+futures = { version = "0.3.28", optional = true }
+
+[features]
+async = ["tokio/full", "dep:futures"]
```

Comment:
I hate to ask for more changes, but because other asynchronous runtimes exist like `async-std`, perhaps we can name this feature (and its module) `tokio`?

Reply:
Done. Thought there would be a naming conflict issue if I named it `tokio`, but it turned out to be fine!